### ISSUE TITLE
How to also keep the types for parameters and not just the names.

### ISSUE THREAD

**Issue by DasOhmoff** [tenfyzhong/CompleteParameter.vim#41]
- Category
    - [x] Question
    - [ ] Bug
    - [ ] Suggestion
- OS
    - [ ] Linux
    - [ ] macOS
    - [x] Windows
    - [ ] Etc.
- Vim
    - [x] Vim
    - [ ] Neovim
- Language
    - [ ] C
    - [x] C++
    - [ ] go
    - [ ] python
    - [ ] rust
    - [ ] javascript
    - [ ] typescript
    - [ ] erlang

# Completion Engine and Commit SHA
YouCompleteMe: f67e5ff27b048d8c55a10ba6a27c6c5b16d0f6ba
(Oct 14, 2018)

First of all I want to say that this plugin is great and really helpful. Thank you!
That beeing said, I wanted to ask if there is a way to keep the types of the parameters when I complete them.

For example instead of:
![grafik](https://user-images.githubusercontent.com/16063186/47620555-88144680-daeb-11e8-8f8b-193cfa321faf.png)

Basically adding the types of the parameters:
![grafik](https://user-images.githubusercontent.com/16063186/47620582-daedfe00-daeb-11e8-83f4-1a798ea56ad8.png)

Is there a way to do this?
If not, what could I do to get this feature somehow?

**Comment by tenfyzhong** [comment]
I don't it is a good idea to keep the types. You can see the function prototype in the command line.

**Comment by DasOhmoff** [comment]
Yes, but the echo gets overwriten by an ycm error message. So I can only see the types for a split second in command line before they get overwritten. Is there a way to fix this?

**Comment by tenfyzhong** [comment]
see #30
